cmd/cortex/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+if [ -n "${CORTEX_ULIMIT_NOFILES:-}" ]; then
+    current_limit=$(ulimit -n)
+    if [ "$current_limit" != "unlimited" ]; then
+        if [ $CORTEX_ULIMIT_NOFILES -gt $current_limit ]; then
+            echo "Setting file description limit to $CORTEX_ULIMIT_NOFILES"
+            ulimit -n $CORTEX_ULIMIT_NOFILES
```

Comment:
ulimit -n modifies the soft limit only.
Go always matches the soft limit to the hard nofile limit. See https://github.com/golang/go/issues/46279. 
Go/cortex runtime will ignore this and use the hard limit